js/piwik.js

```diff
@@ -7180,7 +7197,7 @@ if (typeof window.Piwik !== 'object') {
              * @returns bool
              */
             this.hasRememberedConsent = function () {
-                return !!getCookie('consent');
+                return !!getCookie(CONSENT_COOKIE_NAME);
```

Comment:
@diosmosis maybe here we can simply return ` return !!this.getRememberedConsent()`? to make sure we also handle the case when both cookies are set?

Reply:
sounds good 👍